core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -134,79 +127,18 @@ public void scrollTo(int x, int y) {
     }
 
     @Override
-    protected void onAttachedToWindow() {
-        super.onAttachedToWindow();
-        pagerAdapterObserver.reregister();
+    public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+        scrollingPageChangeListener.onPageScrolled(position, positionOffset, positionOffsetPixels);
```

Comment:
Why isn't the `ScrollingPageChangeListener` implementing the interface itself given it de facto is already? That way we can spare the purely delegating four methods on here.